client.go

```diff
@@ -29,14 +28,325 @@ func main() {
+	}
+	cities.Put(ctx, "1", model.CityAsJson("United Kingdom", "London", 9_540_576))
+	cities.Put(ctx, "2", model.CityAsJson("United Kingdom", "Manchester", 2_770_434))
+	cities.Put(ctx, "3", model.CityAsJson("United States", "New York", 19_223_191))
+	cities.Put(ctx, "4", model.CityAsJson("United States", "Los Angeles", 3_985_520))
+	cities.Put(ctx, "5", model.CityAsJson("Turkey", "Ankara", 5_309_690))
+	cities.Put(ctx, "6", model.CityAsJson("Turkey", "Istanbul", 15_636_243))
+	cities.Put(ctx, "7", model.CityAsJson("Brazil", "Sao Paulo", 22_429_800))
+	cities.Put(ctx, "8", model.CityAsJson("Brazil", "Rio de Janeiro", 13_634_274))
+	mapSize, err := cities.Size(ctx)
+	if err != nil {
+		panic(err)
+	}
+	fmt.Printf("'cities' map now contains %d entries.\n", mapSize)
+	fmt.Println("--------------------")
+}
+
+// sqlExample shows how to work with Hazelcast SQL queries.
+// It maps maps to database tables and runs queries on these maps.
+func sqlExample(client *hazelcast.Client) {
+	sqlService := client.SQL()
+	// Separate examples that shows SQL usage of Hazelcast Go Client
+	createMappingForCapitals(sqlService)
+	clearCapitals(sqlService)
+	populateCapitals(sqlService)
+	selectAllCapitals(sqlService)
+	selectCapitalNames(sqlService)
+}
+
+func createMappingForCapitals(sqlService sql.Service) {
+	// See: https://docs.hazelcast.com/hazelcast/5.1/sql/mapping-to-maps
+	fmt.Println("Creating a mapping...")
+	mappingQuery := fmt.Sprintf(`
+        CREATE OR REPLACE MAPPING capitals TYPE IMap
+		OPTIONS (
+			'keyFormat' = 'varchar',
+			'valueFormat' = 'varchar'
+		)
+	`)
+	ignored, err := sqlService.Execute(context.Background(), mappingQuery)
+	if err != nil {
+		panic(err)
+	}
+	defer ignored.Close()
+	fmt.Println("The mapping has been created successfully.")
+	fmt.Println("--------------------")
+}
+
+func clearCapitals(sqlService sql.Service) {
+	fmt.Println("Deleting data via SQL...")
+	ignored, err := sqlService.Execute(context.Background(), "DELETE FROM capitals")
+	if err != nil {
+		panic(err)
+	}
+	defer ignored.Close()
+	fmt.Println("The data has been deleted successfully.")
+	fmt.Println("--------------------")
+}
+
+func populateCapitals(sqlService sql.Service) {
+	fmt.Println("Inserting data via SQL...")
+	insertQuery := fmt.Sprintf(`
+        INSERT INTO capitals VALUES
+			('Australia','Canberra'),
+            ('Croatia','Zagreb'),
+			('Czech Republic','Prague'),
+			('England','London'),
+			('Turkey','Ankara'),
+			('United States','Washington, DC');
+	`)
+	ignored, err := sqlService.Execute(context.Background(), insertQuery)
+	if err != nil {
+		panic(err)
+	}
+	defer ignored.Close()
+	fmt.Println("The data has been inserted successfully.")
+	fmt.Println("--------------------")
+}
+
+func selectAllCapitals(sqlService sql.Service) {
+	fmt.Println("Retrieving all the data via SQL...")
+	ctx := context.Background()
+	result, err := sqlService.Execute(ctx, "SELECT * FROM capitals")
+	if err != nil {
+		fmt.Errorf("Querying: %w", err)
```

Comment:
The Go convention is starting errors with lower case letters. Same comment for similar code.